weblate/templates/snippets/range-field.html

```diff
@@ -0,0 +1,26 @@
+{% load compress %}
+{% include "django/forms/widgets/input.html" %}
+
+{% compress js %}
+  <script>
+    $(function() {
+      $("input[name={{ widget.name }}]").daterangepicker({
+        autoApply: true,
+        startDate: moment(),
+        endDate: moment(),
+        alwaysShowCalendars: true,
+        opens: "left",
+        ranges: {
+          "Today": [moment(), moment()],
+          "Yesterday": [moment().subtract(1, "days"), moment().subtract(1, "days")],
+          "Last 7 Days": [moment().subtract(6, "days"), moment()],
+          "Last 30 Days": [moment().subtract(29, "days"), moment()],
+          "This Month": [moment().startOf("month"), moment().endOf("month")],
+          "Last Month": [moment().subtract(1, "month").startOf("month"), moment().subtract(1, "month").endOf("month")],
+          "This Year": [moment().startOf("year"), moment().endOf("year")],
+          "Last Year": [moment().subtract(1, "year").startOf("year"), moment().subtract(1, "year").endOf("year")]
+        }
+        }, function(start, end, label) {});
+    });
+  </script>
```

Comment:
Inline script is not allowed by Content-Security-Policy, please place it into `weblate/static/loader-bootstrap.js`.

Reply:
Transferred the JavaScript code into `weblate/static/loader-bootstrap.js`.